src/js/rx/containers/Active.jsx

```diff
@@ -28,20 +33,35 @@ class Active extends React.Component {
     }
   }
 
-  componentDidUpdate() {
-    const newSort = this.props.location.query.sort;
-    const oldSort = this.props.sort;
-
-    if (newSort !== oldSort) {
-      this.props.sortPrescriptions(newSort);
-    }
-  }
-
-  handleSort(sort) {
+  handleSort(sortKey, order) {
+    const sortParam = order === 'DESC' ? sortKey : `-${sortKey}`;
```

Comment:
Is this reversed? I thought descending corresponds to the key with the minus sign.